### DISCUSSION TITLE
Custom data layer name in TileSet

### DISCUSSION THREAD

**Discussion by jonathandw743** [godotengine/godot-proposals#7587] [Editor]
I think it would be useful to include the name of the custom data layer as well as the index.

![Screenshot 2023-08-30 213024](https://github.com/godotengine/godot-proposals/assets/85790952/70847da7-21ca-4730-b919-57edd9f693c6)

As it does in the paint tab.

![Screenshot 2023-08-30 213210](https://github.com/godotengine/godot-proposals/assets/85790952/f6610465-78da-4f4e-8a5b-f3deb69db359)

**Comment by AThousandShips** [comment]
It is a bug and already tracked in: https://github.com/godotengine/godot/issues/76733

Thank you for posting, consolidating there
